### DISCUSSION TITLE
cs_intermission event not firing?

### DISCUSSION THREAD

**Discussion by Rosuav** [saul/demofile#297] [Questions] [answered]
demo.gameEvents.on("cs_intermission", e => console.log("cs_intermission"));

Doesn't ever seem to be called. Is there some other way to get a notification that we've hit half-time? Currently I'm using round_start for round 16 as an indication that every player is on the opposite team now.

**Comment by saul** [comment, marked as answer]
This is expected - only some game events are recorded in demos. See the bottom of [`src/eventtypes.ts`](https://github.com/saul/demofile/blob/master/src/eventtypes.ts) for a complete list.

**Reply by Rosuav** [reply]
I was afraid of that. The string "cs_intermission" does show up in the .dem file, so I was hoping it might show up. Oh well.

**Reply by saul** [reply]
It's listed in the demo as all possible registered events are logged. However only a handful of them are actually recorded in demo files.

**Reply by Rosuav** [reply]
Ahhh, thanks.
